CMarkGFM.hsc

```diff
@@ -73,7 +72,7 @@ extsToLlist (h:t) = do
 
 -- | Resolves a CMarkExtension to its pointer.
 resolveExt :: CMarkExtension -> IO (Maybe ExtensionPtr)
-resolveExt e = do
+resolveExt e = registerPlugins `seq` do
```

Comment:
You're right. I do want to avoid calling the C code behind `registerPlugins` multiple times, which was where I was getting at with this, but I should bury that in `registerPlugins` itself.